src/main/java/com/diamondfire/helpbot/bot/command/impl/other/PollCommand.java

```diff
@@ -0,0 +1,67 @@
+package com.diamondfire.helpbot.bot.command.impl.other;
+
+import com.diamondfire.helpbot.bot.command.argument.ArgumentSet;
+import com.diamondfire.helpbot.bot.command.help.*;
+import com.diamondfire.helpbot.bot.command.impl.Command;
+import com.diamondfire.helpbot.bot.command.permissions.Permission;
+import com.diamondfire.helpbot.bot.events.CommandEvent;
+import net.dv8tion.jda.api.EmbedBuilder;
+
+import java.awt.*;
+import java.util.*;
+
+public class PollCommand extends Command {
+
+
+    @Override
+    public String getName() {
+        return "poll";
+    }
+
+    @Override
+    public HelpContext getHelpContext() {
+        return new HelpContext()
+                .description("Creates a poll.")
+                .category(CommandCategory.OTHER);
+    }
+
+    @Override
+    public ArgumentSet compileArguments() {
+        return new ArgumentSet();
+    }
+
+    @Override
+    public Permission getPermission() {
+        return Permission.DEVELOPER;
+    }
+
+    @Override
+    public void run(CommandEvent event) {
+
+        ArrayList<String> pollOptions = new ArrayList<>();
+        pollOptions.addAll(Arrays.asList(event.getMessage().getContentRaw().split("\\|"))); //poll options + poll question (contains ?poll)
+
+        EmbedBuilder builder = new EmbedBuilder();
+        builder.setTitle(String.valueOf(pollOptions.get(0).subSequence(6,pollOptions.get(0).length()))); //sets title to poll question
```

Comment:
Set pollOptions.get(0) to a variable, and I recommend sending message if there are no options. (Like Invalid Format! Please format your question like Example | Option 1 | Option 2 etc